### PR TITLE
Feature/mysql support & OKD deployment fixes

### PR DESCRIPTION
**Pull request by StyleT**

### CODE REVIEW

File: registry/server/migrations/20191016185102_apps.ts [hunk 1]
```diff
@@ -6,10 +6,10 @@ export async function up(knex: Knex): Promise<any> {
         table.string('name', 50).notNullable().primary();
         table.string('spaBundle', 255).notNullable();
         table.string('cssBundle', 255).nullable();
-        table.json('dependencies').defaultTo('{}');
+        table.json('dependencies');
```
Comment (nightnei) [inline registry/server/migrations/20191016185102_apps.ts]: is it fix for mysql? I think it can cause some errors.

File: registry/Dockerfile [hunk 2]
```diff
@@ -6,7 +6,7 @@ ADD ./ /codebase
 WORKDIR /codebase
 
 RUN npm install --no-package-lock --no-progress --ignore-scripts
-RUN npm install sqlite3
+RUN npm install sqlite3 mysql
```
Comment (nightnei) [inline registry/Dockerfile]: is sqlite needed only for local development? if it's correct, we don't need to install it here, do we?

File: registry/server/routes/config.ts [hunk 3]
```diff
@@ -8,7 +8,7 @@ router.get('/', async (req, res) => {
     const [apps, templates, routes] = await Promise.all([
         knex.select().from('apps'),
         knex.select().from('templates'),
-        knex.select('routes.id as routeId', '*')
```
Comment (oleh-momot) [inline registry/server/routes/config.ts]: image
Does this code work correctly with this change?

Comment (oleh-momot) [reply registry/server/routes/config.ts]: Discussed

**Pull request merged**